locale/id/LC_MESSAGES/django.po

```diff
@@ -181,94 +175,96 @@ msgid "Edit your Profile"
 msgstr "Menyunting Profil"
 
 #: gem/templates/profiles/editprofile.html:12
-#: gem/templates/profiles/viewprofile.html:20
+#: gem/templates/profiles/viewprofile.html:24
 msgid "Update profile"
 msgstr "Menyimpan profil"
 
-#: gem/templates/profiles/register.html:7
+#: gem/templates/profiles/register.html:8
 msgid "Join"
 msgstr "Satukan"
 
-#: gem/templates/profiles/register.html:11
+#: gem/templates/profiles/register.html:12
 msgid "Not a member yet - Join us by signing up below."
 msgstr ""
 "Belum menjadi anggota - Bergabunglah dengan kami dengan mendaftar di bawah "
 "ini."
 
-#: gem/templates/profiles/register.html:20
+#: gem/templates/profiles/register.html:21
 msgid "Choose a username"
 msgstr "Pilih suatu nama pengguna:"
 
-#: gem/templates/profiles/register.html:22
+#: gem/templates/profiles/register.html:23
 msgid "Choose a 4-digit pin"
 msgstr "Pilih 4 digit pin"
 
-#: gem/templates/profiles/register.html:24
+#: gem/templates/profiles/register.html:25
 msgid "JOIN"
 msgstr "Gabung"
 
-#: gem/templates/profiles/register.html:26
+#: gem/templates/profiles/register.html:27
 msgid "already joined?"
 msgstr "Sudah bergabung?"
 
-#: gem/templates/profiles/viewprofile.html:6
-msgid "Hello "
+#: gem/templates/profiles/viewprofile.html:7
+#, fuzzy
```

Comment:
Please delete this line - basically django is confirming that it did a partial match and found these 2 strings to still be the same. If you leave this in, `+#, fuzzy` it ignores this when the translation compiles.